pdc_client/runner.py

```diff
@@ -167,32 +167,13 @@ def run(self, args=None):
         self.client = pdc_client.PDCClient(self.args.server, page_size=self.args.page_size, ssl_verify=ssl_verify)
         try:
             self.args.func(self.args)
-        except beanbag.BeanBagException as exc:
-            self.print_error_header(exc)
-            json = None
+        except beanbag.BeanBagException as ex:
+            print("Server returned following error: [{0}] {1}".format(ex.response.status_code, ex.response.reason), file=sys.stderr)
+            print("Details:", file=sys.stderr)
             try:
-                json = exc.response.json()
-                pretty_print(json, file=sys.stderr)
-            except ValueError:
-                # Response was not JSON
-                print('Failed to parse error response.', file=sys.stderr)
-            except TypeError as e:
-                # Failed to pretty print.
-                print('Failed to correctly display error message. Please file a bug.',
-                      file=sys.stderr)
-                self.logger.info(json, exc_info=e)
```

Comment:
When I try to use a wrong command to get information.

Just execute two times of the command, when I execute the second time, you will get below error with many html codes, I think it is not way to read.
So I think we should keep the ValueError and TypeError parts, what's your thinking @hluk ?
I get below:
```
(pdc) [chcao@dhcp-128-226 pdc_client]$ pdc -s local group-resource-permissions create --group devel --resource arches --permission read
Server returned following error: [500] INTERNAL SERVER ERROR
Details:

<!DOCTYPE html>
<html lang="en">
<head>
  <meta http-equiv="content-type" content="text/html; charset=utf-8">
  <meta name="robots" content="NONE,NOARCHIVE">
  <title>FieldError at /rest_api/v1/auth/group-resource-permissions/</title>
  <style type="text/css">
    html * { padding:0; margin:0; }
    body * { padding:10px 20px; }
    body * * { padding:0; }
    body { font:small sans-serif; }
    body>div { border-bottom:1px solid #ddd; }
    h1 { font-weight:normal; }
    h2 { margin-bottom:.8em; }
    h2 span { font-size:80%; color:#666; font-weight:normal; }
    h3 { margin:1em 0 .5em 0; }
    h4 { margin:0 0 .5em 0; font-weight: normal; }
    code, pre { font-size: 100%; white-space: pre-wrap; }
    table { border:1px solid #ccc; border-collapse: collapse; width:100%; background:white; }
    tbody td, tbody th { vertical-align:top; padding:2px 3px; }
    thead th {
      padding:1px 6px 1px 3px; background:#fefefe; text-align:left;
......
```